Adyen/Model/PrefilledShopperInformation.swift

```diff
@@ -6,6 +6,20 @@
 
 import Foundation
 
+public struct PhoneNumber {
```

Comment:
```suggestion
/// The phone number in E.123 international telephone number notation.
public struct PhoneNumber {
```

Reply:
I have added the description :) 9d6daef2e7cb0abbbe6701564d59fa85bc35c9dc